packages/engine/lib/simulation-control/src/command/create_remove/migration.rs

```diff
@@ -588,11 +618,11 @@ impl<'a> BufferActions<'a> {
                         let maybe_buffer = if i == 0 {
                             ad.null_buffer()
                         } else {
-                            Some(&ad.buffers()[i - 1])
+                            Some(ad.buffer(i - 1))
```

Comment:
`ad` = `agent_data`? Can we rename this as a drive-by so it's more readable? Also applies to other actions below.

Reply:
Thanks, a lot clearer!